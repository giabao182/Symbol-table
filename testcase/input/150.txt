INSERT v number
ASSIGN v 0
BEGIN
INSERT v number
ASSIGN v 1
BEGIN
INSERT v number
ASSIGN v 2
LOOKUP v
END
LOOKUP v
END
LOOKUP v