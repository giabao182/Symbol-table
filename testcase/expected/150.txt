success
success
success
success
success
success
2
1
0